spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/DefaultServerIntrospector.java

```diff
@@ -25,10 +27,11 @@
  * @author Spencer Gibb
  */
 public class DefaultServerIntrospector implements ServerIntrospector {
+	private static final List<Integer> SECURE_PORTS = Arrays.asList(443, 8443);
```

Comment:
I'd agree with that. That's a fine default list. This is only used if discovery hasn't provided a smarter implementation.